src/app/shared/core/post-capture-card/post-capture-card.component.html

```diff
@@ -1,72 +1,76 @@
-      <mat-icon>more_vert</mat-icon>
-    </button>
-  </mat-card-header>
-  <div
-    class="fixed-ratio"
-    [ngStyle]="{
-      height:
-        ratioImg.offsetHeight === ratioImg.offsetWidth
-          ? '100vw'
-          : ratioImg.offsetHeight > ratioImg.offsetWidth
-          ? 'calc(100vw / 4 * 5)'
-          : 'calc(100vw / 16 * 9)'
-    }"
-  >
-    <img
-      #ratioImg
-      id="image"
-      [src]="(asset$ | async)?.sharable_copy"
-      mat-card-image
-    />
-  </div>
-  <mat-card-content>
-    <mat-list dense class="info-list" [class.info-list-show]="openMore">
-      <mat-list-item>
-        <mat-icon class="info-list-icon" mat-list-icon>access_time</mat-icon>
-        <div mat-line>
-          <span class="info-list-title">{{ t('timestamp') }}</span>
-          <span class="info-list-content">{{
-            (asset$ | async)?.uploaded_at | date: 'medium'
-          }}</span>
-        </div>
-      </mat-list-item>
-      <mat-list-item>
-        <mat-icon class="info-list-icon" mat-list-icon>place</mat-icon>
-        <div mat-line>
-          <span class="info-list-title">{{ t('location') }}</span>
-          <span class="info-list-content" mat-line>{{
-            location$ | async
-          }}</span>
-        </div>
-      </mat-list-item>
-      <mat-list-item>
-        <mat-icon
-          class="info-list-img"
-          svgIcon="media-id"
-          mat-list-icon
-        ></mat-icon>
-        <div mat-line>
-          <span class="info-list-title">{{ t('mediaId') }}</span>
-          <span class="info-list-content">{{ (asset$ | async)?.id }}</span>
-        </div>
-      </mat-list-item>
-    </mat-list>
-    <p class="caption-ellipsis mat-body" [class.caption-full]="openMore">
-      {{ (transaction$ | async)?.asset.caption }}
-    </p>
-  </mat-card-content>
-  <mat-card-actions [align]="'end'">
-    <button class="view-all-button" (click)="openMore = !openMore">
-      {{ t('message.viewAll') + '...' }}
-    </button>
-  </mat-card-actions>
-</mat-card>
+<ng-container *ngIf="postCapture$ | async as postCapture">
+  <mat-card *transloco="let t">
+    <mat-card-header>
+      <div mat-card-avatar class="post-capture-avatar"></div>
+      <mat-card-title>{{
+        postCapture.source_transaction.sender
+      }}</mat-card-title>
+      <mat-card-subtitle>{{
+        postCapture.source_transaction.fulfilled_at | date: 'short'
+      }}</mat-card-subtitle>
+      <span class="toolbar-spacer"></span>
+      <button [hidden]="!sharable" (click)="openOptionsMenu()" mat-icon-button>
+        <mat-icon>more_vert</mat-icon>
+      </button>
+    </mat-card-header>
+    <div
+      class="fixed-ratio"
+      [ngStyle]="{
+        height:
+          ratioImg.offsetHeight === ratioImg.offsetWidth
+            ? '100vw'
+            : ratioImg.offsetHeight > ratioImg.offsetWidth
+            ? 'calc(100vw / 4 * 5)'
+            : 'calc(100vw / 16 * 9)'
+      }"
```

Comment:
I think we can remove this `ngStyle` as the ratio of the `sharable_copy` is always 1:1.

Reply:
Great suggestion